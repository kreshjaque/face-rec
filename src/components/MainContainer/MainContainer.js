import React from 'react';
import PropTypes from 'prop-types';
import {MainWrapper} from './MainContainer.style';

const MainContainer = ({ children }) => <MainWrapper>{children}</MainWrapper>;

MainContainer.propTypes = {
  children: PropTypes.oneOfType([
      PropTypes.arrayOf(PropTypes.node),
      PropTypes.node
  ]).isRequired
};

export default MainContainer;
