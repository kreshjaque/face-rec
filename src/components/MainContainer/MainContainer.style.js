import styled from 'styled-components';

export const MainWrapper = styled.main``;

export const Box = styled.div``;
