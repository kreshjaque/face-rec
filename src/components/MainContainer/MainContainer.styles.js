import React from 'react'
import renderer from 'react-test-renderer';
import MainContainer from './MainContainer';

describe('MainContainer', () => {
    const component = renderer.create(<MainContainer><p>dummy test</p></MainContainer>)
    
    test('should render the component', () => {
        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    })
    
})
