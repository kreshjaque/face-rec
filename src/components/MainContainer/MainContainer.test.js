import React from 'react';
import { render } from '@testing-library/react';
import MainContainer from './MainContainer';

describe('MainContainer', () => {
  it('should pass', () => {
    const container = render(<MainContainer />);
    expect(container).toMatchSnapshot();
  });
});
