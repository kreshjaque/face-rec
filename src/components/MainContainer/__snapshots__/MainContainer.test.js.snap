// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer should render the component 1`] = `
<main
  className="sc-bdnxRM"
>
  <p>
    dummy test
  </p>
</main>
`;
